libs/cardano-protocol-tpraos/cardano-protocol-tpraos.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-protocol-tpraos
-version:       1.0.1.0
+version:       1.1.0.0
```

Addition of a `testlib` is not a breaking change. Also the version has already been bumped, since the last released version `1.0.0.0`. So nothing to do here
```suggestion
version:       1.0.1.0
```